doc/ext-pake/api/pake.rst

```diff
@@ -343,9 +346,13 @@ A PAKE primitive is required when constructing a PAKE cipher-suite object, `psa_
 PAKE cipher suites
 ~~~~~~~~~~~~~~~~~~
 
-A PAKE algorithm uses a specific cryptographic primitive for key establishment, specified using a `PAKE primitive <pake-primitive>`. PAKE algorithms also require a cryptographic hash algorithm, which is agreed between the participants.
+Most PAKE algorithms have parameters that must be specified by the application. These parameters include:
+
+*   The cryptographic primitive used for for key establishment, specified using a `PAKE primitive <pake-primitive>`.
```

Comment:
Typo

Reply:
Sorry, I can't seem to see the problem. What am I missing?

Reply:
> The cryptographic primitive used for for key establishment

Small thing

Reply:
Thank you! The hardest to see error in things you have written yourself....